src/content/docs/extras.mdx

```diff
@@ -320,6 +320,215 @@ str_detect("Hello world", "ello")
 </div>
 
   
+## arrow and duckdb
+
+_Fast data storage and database_
+
+One advantage of the learning the `tidyverse` syntax is that it plugs 
+in seemlessly with databases using the 
+[dbplyr](https://dbplyr.tidyverse.org/) 
+package. The **arrow** package provides a really fast and memory efficient data 
+storage format (`.parquet`). The **duckdb** package provides a really fast and 
+memory efficient database system. For example, imagine you have a 10M row 
+dataset with hundreds of variables. You want to take group-averages of a few 
+varaibles. Arrow/duckdb will do this *astonigshly* fast by not having to load 
+an entire CSV into memory.
+
+For this example, we will create a small dataset using the following code:
+```r
+data(airquality)
+dir.create("airquality")
+arrow::write_dataset(
+  airquality, 
+  path = "airquality",
+  partitioning = c("Month") # <-- NB: partition your data by the main grouping vars for efficiency
+)
+```
+
+For example using a massive NYC Taxi dataset, see https://arrow.apache.org/docs/2.0/r/articles/dataset.html. The file is 37 gigabytes (bigger than most people's RAM) and arrow can do a `group_by()` and `summarize()` in a few seconds on a standard macbook. 
+
+#### Import/export data with arrow
+
+```r
+airquality = arrow::open_dataset("airquality")
```

Comment:
One other thing: I'd call this `aq` to avoid confusion with the base `airquality` R data.frame. It similarly helps to head off confusion about is held in R's memory versus what is a pointer/reference to some external data.